File: problemas/geometria-computacional/2840/2840.go
package main

import (
    "fmt"
)

func main() {
    var r, l int64

    fmt.Scanf("%d %d", &r, &l)

    pi := 3.1415
    v := (4.0 * pi * float64(r) * float64(r) * float64(r)) / 3.0

    fmt.Printf("%d\n", int(float64(l)/v))
}

File: problemas/geometria-computacional/2840/2840.c
#include <stdio.h>

int main() {
  int R, L;
  double pi = 3.1415;

  scanf("%d %d", &R, &L);

  double V = (4.0 * pi * R * R * R) / 3.0;
  printf("%d\n", (int)(L / V));

  return 0;
}